.github/ISSUE_TEMPLATE/new-language.md

```diff
@@ -0,0 +1,22 @@
+---
+name: new language
+about: Request a new language for onefetch to support
+title: "[NEW LANGUAGE]"
+labels: 'Type: Enhancement, good first issue'
+assignees: ''
```

Comment:
This line can be removed ?

Reply:
:thinking: I'm not sure, it was automatically added by GitHub when I finished making the template.